lib/modules/hosts/vidgif.js

```diff
@@ -1,15 +1,15 @@
 export default {
-	moduleID: 'swirl',
-	name: 'swirl',
-	domains: ['swirl.xyz'],
+	moduleID: 'vidgif',
```

Comment:
We shouldn't change the moduleID since it's used as the settings key (`name` is what's displayed anyways).

Maybe also add `// vidgif.com was originally swirl.xyz` so we remember not to change it in the future.